client/src/components/SessionManagerField.js

```diff
@@ -1,20 +1,15 @@
 /* global window */
-import React, { Component } from 'react';
+import React from 'react';
+import LoginSession from './LoginSession';
 
-class SessionManagerField extends Component {
-  render() {
+function SessionManagerField(props) {
     return (
-      <div>
-        <p>hello world</p>
+      <div className={'SessionManagerField'}>
+        {props.loginSessions.map((loginSession) =>
+          <LoginSession key={loginSession.ID} {...loginSession} />
```

Comment:
Do we actually need to set the key attr here?

Reply:
Yep, React throws errors if you don't on a list.